unlock-app/.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     browser: true,
     jest: true,
   },
-  plugins: ['jest', 'mocha', 'promise'],
+  plugins: ['jest', 'mocha', 'promise', 'import'],
```

Comment:
The changes here allow eslint to correctly resolve imports of TS files. Without this, you'd get a "module not found" error in any file that imported some TS.